qp_klp/Step.py

```diff
@@ -607,6 +613,58 @@ def load_preps_into_qiita(self, qclient):
 
         return df
 
+    def _load_preps_into_qiita2(self, qclient, prep_id, qiita_id, out_dir,
+                               project):
+        surl = f'{qclient._server_url}/study/description/{qiita_id}'
+        prep_url = (f'{qclient._server_url}/study/description/'
+                    f'{qiita_id}?prep_id={prep_id}')
+
+        if self.pipeline.pipeline_type == Step.AMPLICON_TYPE:
+            files = self._get_files_amplicon(out_dir, project)
+        else:
+            atype = 'per_sample_FASTQ'
+            files = self._get_files_meta(out_dir, project, prep_id)
+
+        for f_type in files:
+            if not files[f_type]:
+                # if one or more of the expected list of reads is empty,
+                # raise an Error.
+                raise ValueError(f"'{f_type}' is empty")
+
+        # ideally we would use the email of the user that started the SPP
+        # run but at this point there is no easy way to retrieve it
+        pdata = {'user_email': 'qiita.help@gmail.com',
+                 'prep_id': prep_id,
+                 'artifact_type': atype,
```

Comment:
It looks like this will produce a `NameError` if it's amplicon data

Reply:
I reviewed the earlier versions of the code that handle loading for amplicon data and can't find a discrepancy in the parameters. Do you recall why you thought this? In either case I'll add a fresh amplicon test to the round of testing, once loading is re-enabled.

Reply:
You previously used a pattern like:

```python
if foo == bar:
    atype = 'thing'
```

where the variable `atype` was only defined if a specific branch was taken, which would lead to a `NameError`

Reply:
thank you for fixing it

Reply:
I think it was always like that but all good either way. Perhaps the rewrite to handle meta*omic and amplicon both w/replication and w/out took care of it. No worries. Ty for the review.